index.js

```diff
@@ -1,11 +1,15 @@
 const beautify = require('pretty')
 
+const isHtmlString = received => typeof received === 'string' && received[0] === '<'
+const isVueWrapper = received => typeof received === 'object' && received.isVueComponent
```

Comment:
This property will soon be renamed to isVm. Instead use the public method `isVueInstance()`